x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/cytoscape.tsx

```diff
@@ -98,11 +108,24 @@ export function Cytoscape({ children, elements, height, style, width }: Cytoscap
   // Trigger a custom "data" event when data changes
   useEffect(() => {
     if (cy) {
-      cy.add(elements);
+      if (itemsDeleted === false) {
+        cy.add(elements);
+      } else if (itemsDeleted === true) {
```

Comment:
`itemsDeleted` is a `boolean` type, so `else if` is an overkill 
```suggestion
      if (itemsDeleted === false) {
        cy.add(elements);
      } else {
```

Reply:
Updated 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9